src/main/java/com/iexec/worker/pubsub/SubscriptionService.java

```diff
@@ -84,16 +84,25 @@ public void subscribeToTopic(String chainTaskId) {
     /**
      * Unsubscribe from topic if already subscribed.
      *
-     * @param chainTaskId
+     * @param chainTaskId id of the task to unsubscribe and purge
+     * @return true if unsubscribed and purge topic successfully, false otherwise
      */
-    public void unsubscribeFromTopic(String chainTaskId) {
+    @Override
+    public boolean purgeTask(String chainTaskId) {
```

Comment:
```suggestion
    public boolean purgeTask(final String chainTaskId) {
        log.debug("purgeTask [chainTaskId:{}]", chainTaskId);
```

Reply:
Resolve in [f0efa1b](https://github.com/iExecBlockchainComputing/iexec-worker/pull/620/commits/f0efa1b0cee87e905e79617f53da621afa995a91)